osu.Game.Tournament/JsonPointConverter.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Diagnostics;
+using System.Drawing;
+using Newtonsoft.Json;
+
+namespace osu.Game.Tournament
+{
+    /// <summary>
+    /// We made a change from using SixLabors.ImageSharp.Point to System.Drawing.Point at some stage.
+    /// This handles converting to a standardised format on json serialize/deserialize operations.
+    /// </summary>
+    internal class JsonPointConverter : JsonConverter<Point>
+    {
+        public override void WriteJson(JsonWriter writer, Point value, JsonSerializer serializer)
+        {
+            // use the format of LaborSharp's Point since it is nicer.
+            serializer.Serialize(writer, new SixLabors.ImageSharp.Point(value.X, value.Y));
```

Comment:
Maybe slightly nicer to use an anonymous type to not have to reference imagesharp at all?

```suggestion
            serializer.Serialize(writer, new { value.X, value.Y });
```

Output is the same as far as I can see.